core/src/main/kotlin/model/Documentable.kt

```diff
@@ -70,19 +80,24 @@ class Parameter(
     val type: TypeWrapper,
     override val actual: List<PlatformInfo>,
     override val extra: MutableSet<Extra> = mutableSetOf()
-) : Documentable() {
+) : Documentable(), WithVisibility {
     override val children: List<Documentable>
         get() = emptyList()
+
+    override val visibility: Visibility = Visibilities.PUBLIC
 }
 
 interface PlatformInfo {
     val documentationNode: DocumentationNode
     val platformData: List<PlatformData>
+    val descriptor: DeclarationDescriptor?
 }
 
 class BasePlatformInfo(
     override val documentationNode: DocumentationNode,
-    override val platformData: List<PlatformData>) : PlatformInfo {
+    override val platformData: List<PlatformData>,
+    override val descriptor: DeclarationDescriptor?
```

Comment:
If we are to store the descriptor there, it should be a `List` or better yet a `Map<PlatformData, List<DeclarationDescriptor>> `

Reply:
I've moved this information to cache in plugin.